### DISCUSSION TITLE
Implement Client-Side Partial Updates in Fresh

### DISCUSSION THREAD

**Discussion by m7medVision** [TeaByte/NakhlahJS#53] [Ideas]
I'd like to open a discussion about adding a new feature to our Fresh app: **client-side partial updates**, also known as [ partials](https://fresh.deno.dev/docs/concepts/partials). This capability would allow sections of a page to be refreshed directly by the server without requiring a full page reload. 

Let's discuss the potential benefits and trade-offs of implementing partials in our app.

**Comment by TeaByte** [comment]
I did not know that there is partials on Fresh framework
Certainly using it will make a big change

**Comment by m7medVision** [comment]
### I believe we can avoid reloading the Navbar, web VS Code editor, and what I highlight it in the photo. However, I'm concerned that not reloading VS Code or the relevant section might cause issues for testing.
![image](https://github.com/TeaByte/NakhlahJS/assets/88824957/ac18efd8-3b84-46cc-9ee1-2fb43222bcbc)

**Reply by m7medVision** [reply]
@TeaByte

**Reply by TeaByte** [reply]
The Navbar avoid reloading will be great and will optimize the site more
